src/index.html

```diff
@@ -11,6 +11,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+        class="form__field"
```

Comment:
```suggestion
      class="form__field"
```
To much spaces:
![image](https://user-images.githubusercontent.com/99652389/178439356-a1ed21e5-10dd-4e66-a189-7260f378e20d.png)